### DISCUSSION TITLE
Neovim + Premake5?

### DISCUSSION THREAD

**Discussion by that404nerd** [premake/premake-core#2126] [Q&A]
Hi there! I started using neovim recently and use c++ as my primary language. But there is no information related to integrating neovim with premake5. This is causing a lot of problems for me. Any help is greatly appreciated!

**Comment by nickclark2016** [comment]
Just to not leave this discussion unresolved, it's not something that I personally have attempted to integrate.  If it's trivial to integrate, we'd certainly welcome documentation PRs on how to do it. If it needs separate functionality for integration of Premake and Neovim, we'd probably want to see a module for that.
